--- sdlmame0134u2/makefile~	2009-10-07 12:12:04.000000000 +0200
+++ sdlmame0134u2/makefile	2009-10-07 12:14:42.000000000 +0200
@@ -330,19 +330,16 @@
 CCOMFLAGS += -pg
 endif
 
-# add the optimization flag
-CCOMFLAGS += -O$(OPTIMIZE)
-
 # if we are optimizing, include optimization options
 # and make all errors into warnings
 ifneq ($(OPTIMIZE),0)
 ifneq ($(TARGETOS),os2)
 ifndef IA64
-CCOMFLAGS += -Werror -fno-strict-aliasing $(ARCHOPTS)
+CCOMFLAGS += $(ARCHOPTS)
 else
 endif
 else
-CCOMFLAGS += -fno-strict-aliasing $(ARCHOPTS)
+CCOMFLAGS += $(ARCHOPTS)
 endif
 endif
 
