--- sdlmame0133u1/makefile~	2009-06-11 06:38:13.000000000 +0200
+++ sdlmame0133u1/makefile	2009-08-12 12:29:55.000000000 +0200
@@ -570,15 +570,15 @@
 
 $(OBJ)/%.o: $(SRC)/%.c | $(OSPREBUILD)
 	@echo Compiling $<...
-	$(CC) $(CDEFS) $(CFLAGS) -c $< -o $@
+	$(CC) $(CDEFS) $(OPTFLAGS) $(CFLAGS) -c $< -o $@
 
 $(OBJ)/%.pp: $(SRC)/%.c | $(OSPREBUILD)
 	@echo Compiling $<...
-	$(CC) $(CDEFS) $(CFLAGS) -E $< -o $@
+	$(CC) $(CDEFS) $(OPTFLAGS) $(CFLAGS) -E $< -o $@
 
 $(OBJ)/%.s: $(SRC)/%.c | $(OSPREBUILD)
 	@echo Compiling $<...
-	$(CC) $(CDEFS) $(CFLAGS) -S $< -o $@
+	$(CC) $(CDEFS) $(OPTFLAGS) $(CFLAGS) -S $< -o $@
 
 $(OBJ)/%.lh: $(SRC)/%.lay $(FILE2STR)
 	@echo Converting $<...
